piccolo_api/change_password/endpoints.py

```diff
@@ -135,19 +139,23 @@ async def post(self, request: Request) -> Response:
         # after password changes we invalidate session and redirect user
         # to login endpoint to login again with new password
         response = RedirectResponse(
-            url="/login/", status_code=HTTP_303_SEE_OTHER
+            url=self._redirect_to, status_code=HTTP_303_SEE_OTHER
         )

```

Comment:
I think this should be removed because we don't want user be able to change the redirect (by default it should be `redirect_to ="/login"` like it was before) because we are invalidating the session cookie and the user **MUST** log in again. I intentionally did not add `_redirect_to` property.

Reply:
I see your point. The only issue is if someone has their login at a different URL. Instead of `redirect_to`, it could be `login_url` or something?

Reply:
Well noticed! You are right. I didn't think anyone could set a different login url than` /login`. `login_url` will be great and understandable.